purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEvent.kt

```diff
@@ -0,0 +1,61 @@
+package com.revenuecat.purchases.customercenter.events
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.utils.serializers.DateSerializer
+import com.revenuecat.purchases.utils.serializers.UUIDSerializer
+import dev.drewhamilton.poko.Poko
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Type representing a customer center event and associated data. Meant for RevenueCatUI use.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+@Poko
+@Serializable
+class CustomerCenterEvent(
+    val creationData: CreationData,
+    val eventData: Data,
+) {
+
+    companion object {
+        val json = Json.Default
+
+        fun fromString(string: String): CustomerCenterEvent {
```

Comment:
This also could be `internal` I think? I'm not sure if it's actually used though...